packages/react-dialog/Spec.md

```diff
@@ -2,62 +2,302 @@
+
+#### v8
+
+In v8 there are Dialog and Modal components which are relevant to the Dialog component for v9. The Dialog component was intended to be used primarily for confirming actions, whereas Modal component was intended to be used for lengthy content that may contain forms and other controls. This spec will only cover the comparison to the Dialog component.
+
+The visibility of the dialog is controlled through the `hidden` prop whose its value should be a react state boolean provided from the consumer.
+
+[Documentation for v8 Dialog](https://developer.microsoft.com/en-us/fluentui#/controls/web/dialog)
+
+Sample code:
+
+```jsx
+  <DefaultButton secondaryText="Opens the Sample Dialog" onClick={toggleHideDialog} text="Open Dialog" />
+
+  <Dialog
+    hidden={hideDialog}
+    onDismiss={toggleHideDialog}
+    dialogContentProps={dialogContentProps}
+    modalProps={modalProps}
+  >
+    <DialogFooter>
+      <PrimaryButton onClick={toggleHideDialog} text="Send" />
+      <DefaultButton onClick={toggleHideDialog} text="Don't send" />
+    </DialogFooter>
+  </Dialog>
+
+```
+
+#### v0
+
+In v0, the Dialog component expects all the content through props, including the content, actions etc. The dialog component uses the `trigger` prop that expects a React component to control its visibility. The element passed to this prop will be rendered in-place where the dialog is defined.
+
+[Documentation for v0 Dialog](https://fluentsite.z22.web.core.windows.net/components/dialog/definition)
+
+```jsx
+<Dialog
+  cancelButton="Connect protocol"
+  confirmButton="Transmit capacitor"
+  content="Connect driver"
+  header="Transmit capacitor"
+  headerAction="Generate protocol"
+  trigger={<Button content="A trigger" />}
+/>
+```
+
+## API proposal
+
+Dialog component is composed by 3 sub-components: Header, Body and Footer.
+
+### Components
+
+| Component    | Purpose                                                  |
+| ------------ | -------------------------------------------------------- |
+| Dialog       | The main wrapper component.                              |
+| DialogHeader | (optional) Component for the title and the close button. |
+| DialogBody   | (optional) Component for the main content of dialog.     |
+| DialogFooter | (optional) Component for the main actions of dialog.     |
+
+## Dialog
+
+The dialog is a container which handles styling (border, background etc.) and the overlay (dimmed background).
+
+### Anatomy
+
+![Visual anatomy of Dialog component](./assets/dialog-anatomy.png)
+
+### API
+
+| Property    | Values                        | Default   | Purpose                                    |
+| ----------- | ----------------------------- | --------- | ------------------------------------------ |
+| type        | `modal`, `non-modal`, `alert` | `modal`   | Dialog variations                          |
+| isOpen      | boolean                       | `false`   | Set to `true` when the dialog is visible   |
```

Comment:
not 100% sure what our approach regarding naming boolean props is. If there is a standard that it should start with `is*` prefix. 

If not I'd consider to use just `open` to follow more HTML standards

Reply:
Yes I agree, changing it to `open`. Thanks ^_^